src/components/wallet/methods/depositAsset.ts

```diff
@@ -57,11 +57,31 @@ export default async function depositAsset(
     this.logger.response(tomlURL.toString(), tomlText)
     const toml = TOML.parse(tomlText)
 
+    this.logger.instruction(
+      `Received WEB_AUTH_ENDPOINT from TOML: ${toml.WEB_AUTH_ENDPOINT}`
+    )
+    this.logger.instruction(
+      `Received TRANSFER_SERVER from TOML: ${toml.TRANSFER_SERVER}`
+    )
+    this.logger.instruction(
+      `Received asset issuer from TOML: ${toml.SIGNING_KEY}`
+    )
+    this.logger.instruction(
+      'Check /info endpoint to see if we need to authenticate'
+    )
     const infoURL = `${toml.TRANSFER_SERVER_SEP0024}/info`
     this.logger.request(infoURL)
     const info = await axios.get(infoURL).then(({ data }) => data)
     this.logger.response(infoURL, info)
-    console.log(info)
+
+    this.logger.instruction(
+      'Deposit is enabled, and requires authentication so we should go through SEP-0010'
+    )
+    this.logger.instruction(
+      'Start the SEP-0010 flow to authenticate the wallet’s Stellar account'
+    )
+    const params = { account: this.account.publicKey }
```

Comment:
im a little confused here as these params dont seem to be used anymore, but they definitely should be.  It looks like our sep10 request is missing the account param im actually surprised the existing code works at all

Reply:
They're used on the next line in `this.logger.request(this.toml.WEB_AUTH_ENDPOINT, params)` - is that what you meant? And the `this.account.publicKey` is used in the next network call to the `WEB_AUTH_ENDPOINT` (let me know if I'm misunderstanding the comment!)

Reply:
Oh no i just missed that github folded the two relevent lines down so i missed them.  This makes sense.  It would be nice to use the params object you define here inside the params of the axios call just so we ensure that params always stay in sync and reduce duplication